Cannot compile v0.1.0

Cannot find <keyboard.h>

=======Log=======
$ make run-iso
Scanning dependencies of target arch
[  9%] Building ASM object arch/x86/CMakeFiles/arch.dir/boot/boot.S.o
[ 18%] Building CXX object arch/x86/CMakeFiles/arch.dir/kernel/cpuinfo.cpp.o
[ 27%] Building CXX object arch/x86/CMakeFiles/arch.dir/kernel/flag.cpp.o
[ 36%] Building CXX object arch/x86/CMakeFiles/arch.dir/kernel/idt.cpp.o
/home/hhro/develop/lets_make_kernel/arch/x86/kernel/idt.cpp:2:10: fatal error: keyboard.h: No such file or directory
 #include <keyboard.h>
          ^~~~~~~~~~~~
compilation terminated.
arch/x86/CMakeFiles/arch.dir/build.make:126: recipe for target 'arch/x86/CMakeFiles/arch.dir/kernel/idt.cpp.o' failed
make[3]: *** [arch/x86/CMakeFiles/arch.dir/kernel/idt.cpp.o] Error 1
CMakeFiles/Makefile2:224: recipe for target 'arch/x86/CMakeFiles/arch.dir/all' failed
make[2]: *** [arch/x86/CMakeFiles/arch.dir/all] Error 2
CMakeFiles/Makefile2:181: recipe for target 'CMakeFiles/run-iso.dir/rule' failed
make[1]: *** [CMakeFiles/run-iso.dir/rule] Error 2
Makefile:157: recipe for target 'run-iso' failed
make: *** [run-iso] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ set(CMAKE_TRY_COMPILE_TARGET_TYPE "STATIC_LIBRARY")
 project(lets_make_kernel
     DESCRIPTION "Lets make kernel!"
     LANGUAGES   CXX
-    VERSION     0.1.0
+    VERSION     0.1.1
 )
 
 if(NOT ARCH)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,7 +24,7 @@ set(CMAKE_ASM_COMPILER_TARGET ${TRIPLE})
 
 enable_language(ASM)
 
-include_directories(kernel)
+include_directories(kernel/include)
 include(arch/${ARCH}/defconfig.cmake)
 
 add_subdirectory("arch/${ARCH}")

